api/eoa_test_account.go

```diff
@@ -0,0 +1,78 @@
+package api
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"io"
+	"math/big"
+
+	gethCommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	gethCrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/onflow/flow-go/fvm/evm/emulator"
+)
+
+// address:  658bdf435d810c91414ec09147daa6db62406379
+const eoaTestAccount1KeyHex = "9c647b8b7c4e7c3490668fb6c11473619db80c93704c70893d3813af4090c39c"
```

Comment:
maybe better to use the configured evm account, better than have random keys hardcoded.